.github/workflows/lint.yaml

```diff
@@ -43,6 +43,8 @@ jobs:
           persist-credentials: false
       - name: Setup Python
         uses: actions/setup-python@v4
+        with:
+          python-version: '^3.8.x'
```

Comment:
Hmm. I think we were aiming for Python 3.7 as the baseline. Oscar ran into various issues between .7 and .8, we should keep it consistent.

See https://github.com/youtube/cobalt/blob/main/.github/workflows/pytest.yaml#L21

Reply:
@andrewsavage1, you recommended 3.8.x, any objections if we go with 3.7.x?

Reply:
That sounds okay

Reply:
Changed to 3.7.x